linetools/spectralline.py

```diff
@@ -266,7 +267,6 @@ def setz(self, z):
         if not isinstance(z,float):
             raise IOError("Input redshift needs to be a float")
         # Set
-        self.attrib['z'] = z
         self.limits._z = z
         # Warning?
         if self.limits.is_set():
```

Comment:
This allows you to modify the redshift of the line
without understanding how to use LineLimit